File: wlxx-services/wlxx-system/wlxx-system-client/src/main/java/com/xzg/wlxx/system/client/entity/po/Emp.java
package com.xzg.wlxx.system.client.entity.po;


import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableName;
import com.xzg.wlxx.common.base.BasePo;
import com.xzg.wlxx.system.client.enums.SexEnum;
import lombok.Data;
import lombok.EqualsAndHashCode;

@TableName("emp")
@Data
@EqualsAndHashCode(callSuper = true)
public class Emp extends BasePo<Emp> {

    @TableField("emp_code")
    private String empCode;

    @TableField("emp_name")
    private String empName;

    @TableField("sex")
    private SexEnum sex;
}

File: wlxx-services/wlxx-system/wlxx-system-client/src/main/kotlin/com/xzg/wlxx/system/entity/po/Emp.kt
package com.xzg.wlxx.system.entity.po

import com.baomidou.mybatisplus.annotation.TableField
import com.baomidou.mybatisplus.annotation.TableName
import com.xzg.wlxx.common.base.BasePo
import com.xzg.wlxx.system.entity.enums.SexEnum

@TableName("emp")
data class Emp(
    @TableField("emp_code")
    var empCode: String? = null,

    @TableField("emp_name")
    var empName: String? = null,

    @TableField("sex")
    var sex: SexEnum? = null
) : BasePo<Emp>()